locale/en/blog/weekly-updates/weekly-update.2016-04-23.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Weekly Update - Apr 23rd, 2016
+author: Minwoo Jung (github.com/JungMinu)
+date: 2016-04-23T12:00:00.000Z
+status: publish
+category: weekly
+slug: weekly-update-2016-04-23
+layout: blog-post.hbs
+---
+
+### Node.js News
+Node v5.11.0 (Stable) is released.
```

Comment:
I'd remove lines 11 and 12 as the only news item is repeated prominently in line 14.